tests/test_emulators.py

```diff
@@ -1060,3 +1072,142 @@ def test_simple(self):
         np.testing.assert_almost_equal(
             hbvec.diagnostics["DIAG_NASH_SUTCLIFFE"], d["DIAG_NASH_SUTCLIFFE"], 4
         )
+
+
+class TestRouting:
+    def test_lievre_tutorial(self):
+        """
+        This test reproduces the Lievre tutorial setup:
+
+        http://raven.uwaterloo.ca/files/RavenTutorial6.zip
+
+        """
+
+        def to_py_datetime(x_dt):
+            """
+            Cast an xarray datetime to a python datetime.
+
+            Parameters
+            ----------
+            x_dt: xarray.core.accessor_dt.DatetimeAccessor
+
+            Returns
+            -------
+            datetime.datime
+
+            """
+            return dt.datetime(
+                x_dt.year.item(),
+                x_dt.month.item(),
+                x_dt.day.item(),
+                x_dt.hour.item(),
+                x_dt.minute.item(),
+                x_dt.second.item(),
+            )
+
+        ###############
+        # Input files #
+        ###############
+
+        routing_product_shp_path = get_local_testdata(
+            "raven-routing-sample/finalcat_hru_info.zip"
+        )
+        vic_streaminputs_nc_path = get_local_testdata(
+            "raven-routing-sample/VIC_streaminputs.nc"
+        )
+        vic_temperatures_nc_path = get_local_testdata(
+            "raven-routing-sample/VIC_temperatures.nc"
+        )
+
+        #########
+        # Model #
+        #########
+
+        model = Routing()  # "/tmp/ravenpy_routing_emu_dev")
+
+        #######
+        # RVI #
+        #######
+
+        streaminputs = xr.open_dataset(vic_streaminputs_nc_path)
+
+        start_date = streaminputs.time[0].dt
```

Comment:
streaminputs.indexes['time'][0] might give you something easier to work with.

Reply:
thanks!